prometheus_api_client/prometheus_connect.py

```diff
@@ -64,6 +64,19 @@ def __init__(
         self._session = requests.Session()
         self._session.mount(self.url, HTTPAdapter(max_retries=retry))
 
+    def check_prometheus_connection(self, params: dict = None) -> bool:
+        """Check database metrics (Prometheus) connection."""
```

Comment:
Can you add a couple of lines describing the input and output? (Similar to [this](https://github.com/AICoE/prometheus-api-client-python/blob/494e3976d0835c07d5c9f1c476462ba3588b3130/prometheus_api_client/prometheus_connect.py#L81-L91))

Reply:
Added more detailed description! Thanks!